draft-barnes-acme.md

```diff
@@ -754,7 +754,17 @@ The values provided in the CSR are only a request, and are not guaranteed.  The
 
 It is up to the server's local policy to decide which names are acceptable in a certificate, given the authorizations that the server associates with the client's account key.  A server MAY consider a client authorized for a wildcard domain if it is authorized for the underlying domain name (without the "*" label).  Servers SHOULD NOT extend authorization across identifier types.  For example, if a client is authorized for "example.com", then the server should not allow the client to issue a certificate with an iPAddress subjectAltName, even if it contains an IP address to which example.com resolves.
 
-If the CA decides to issue a certificate, then the server returns the certificate in a response with status code 201 (Created).  The server MUST indicate a URL for this certificate in a Location header field.
+If the CA decides to issue a certificate, then the server creates a new certificate resource and returns a URI for it in the Location header field of a 201 (Created) response.
+
+~~~~~~~~~~
+
+HTTP/1.1 201 Created
+Location: https://example.com/acme/cert/asdf
+Retry-After: 120
+
+~~~~~~~~~~
+
+If the CA has not yet issued the certificate, the body of this response will be empty, and MUST include a Retry-After header to indicate when the server believes the certificate will be issued (as in the example above).  After that time, the client may send a GET request to the certificate URI in order to get the certificate.
```

Comment:
I was afraid you would say that.  The alternative is to have the client follow up with a GET to the cert URI, resulting in a 202 + Retry-After.  It just seemed like this would optimize out the GET, but I think we actually need it.